boards/arm/nucleo_h745zi_q/nucleo_h745zi_q_m4.dts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/h7/stm32h745Xi_m4.dtsi>
+#include "nucleo_h745zi_q.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32H745ZI-NUCLEO board";
+	compatible = "st,stm32h745zi-nucleo", "st,stm32h745";
+
+	/* HW resources are split between CM7 and CM4 */
+	chosen {
+		/* zephyr,console = &usart3; */
+		/* zephyr,shell-uart = &usart3; */
+		zephyr,sram = &sram1;
+		zephyr,flash = &flash1;
+	};
+
+	aliases {
+		led0 = &green_led;
+		led1 = &blue_led;
+		pwm-led0 = &red_pwm_led;
+		sw0 = &user_button;
+	};
+};
+
+&usart3 {
+	current-speed = <115200>;
+	/* status = "okay"; */
+};
+
+&uart8 {
+	current-speed = <115200>;
+	/* status = "okay"; */
```

Comment:
I think that these comments should not be here. Just don't use status (will remain disabled) or remove the entire entry.

Reply:
Changed M4 to use uart8 and M7 rely on usart3

Reply:
Why is it still commented if M7 does not rely on it?

Reply:
> Why is it still commented if M7 does not rely on it?

Should I uncomment both lines?

`/* zephyr,console = &uart8; */`
`/* zephyr,shell-uart = &uart8; */`

Reply:
> Why is it still commented if M7 does not rely on it?

@erwago Why? I think, it is better to ask you (@erwango ), since it was change proposed by you.